src/Quaternion_TEST.cc

```diff
@@ -252,6 +252,8 @@ TEST(QuaternionTest, Math)
   q.Normalize();
   EXPECT_TRUE(q == math::Quaterniond(0.182574, 0.365148, 0.547723, 0.730297));
 
+  math::Quaterniond q2 = q.Normalized();
```

Comment:
Isn't `q` already normalized in this line? So in this case `q2 == q`, right?

How about moving this test case above line 252, so that the vector returned in the `Normalized` call is different from the original vector?

Reply:
Yes, sure, I'll do it!

Reply:
@chapulina Done! :wink: